workinggroups/README.md

```diff
@@ -0,0 +1,11 @@
+# CNCF Working Groups
+
+## Introduction
+
+The purpose of working groups are to study and report on a particular question and make recommendations based on its findings. The end result of a working group may be a new project proposal, landscape, whitepaper or even a report detailing their findings. The intention of working groups is not to host a full project or specification. Working Groups can be formed at any time but must be sponsored by a TOC member and voted with a super majority vote by the CNCF TOC. The TOC can also shut down a working group with a super majority vote.
```

Comment:
"The intention of working groups is not to host a full project or specification."  What does this mean for the CloudEvents stuff?